pyproject.toml

```diff
@@ -45,14 +45,15 @@ bokeh = "^2.2"
 panel = "^0.12"
 xarray = "^0.19"
 netCDF4 = "^1.5"
+llvmlite = ">=0.37"
 datashader = "^0.13"
-ruyaml = "^0.20.0"
+"ruamel.yaml" = "^0.17.0"
 geoviews = "^1.9.2"
 Cartopy = "<0.20"  # due to https://github.com/holoviz/geoviews/issues/529
 Shapely = "<1.8"
 
 [tool.poetry.dev-dependencies]
-black = "^21.11b1"
+black = "^21.11b0"
```

Comment:
What is the problem with 22.11b1? Or is it not available in conda-forge?

Reply:
Indeed this is latest on conda-forge at least for OSX